ipaserver/plugins/realmdomains.py

```diff
@@ -118,10 +128,22 @@ class realmdomains(LDAPObject):
     )
 
 
-
 @register()
 class realmdomains_mod(LDAPUpdate):
-    __doc__ = _('Modify realm domains.')
+    __doc__ = _("""
+    Modify realm domains
+
+    DNS check: When adding domain to the list manually, a DNS check is
+    performed by default. It checks whether the domain is associated
+    with the IPA realm. In practice it checks whether the domain has
+    _kerberos TXT record containing IPA realm name. This check can be
+    skipped by specifying --force option.
```

Comment:
nitpicking, but I would rephrase:
When manually adding a domain to the list, a DNS check is performed by default. It ensures that the domain is associated with the IPA realm, by checking whether the domain has a _kerberos TXT record containing the IPA realm name. This check can be skipped by specifying --force option.

Reply:
This got me to another point: Active Directory domains never have _kerberos TXT record associated, so the check is inadequate for that use case. We should use a different method by looking up for MSDCS entries in the domain and its parents. However, I'm not sure where we should stop. See https://technet.microsoft.com/en-us/library/cc759550(WS.10).aspx for details